ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -359,9 +359,11 @@ public void storeStartupTests() throws IOException, StoreException {
    */
   @Test
   public void basicTest() throws InterruptedException, IOException, StoreException {
-    // PUT a key that is slated to expire when time advances by 2ms
-    MockId addedId = put(1, PUT_RECORD_SIZE, time.milliseconds() + 1).get(0);
-    time.sleep(2);
+    // PUT a key that is slated to expire when time advances by 1s
+    // since index loses precision to int for expiration time, move time to atleast by a second
+    time.sleep(Time.MsPerSec);
```

Comment:
it is not strictly required. Do you remember you sleep for 1 ms at the beginning of set up for expiration purposes, this sleep is for the same reason. Removed sleep here. But instead of sleeping for 1 ms in set up, sleeping for 1 sec now.